projects/storefrontapp-e2e-cypress/cypress/helpers/order-cancellations-returns.ts

```diff
@@ -0,0 +1,33 @@
+export function checkTabs() {
+  cy.get('cx-tab-paragraph-container > h3')
+    .eq(1)
+    .should('contain', 'RETURNS')
+    .click();
+
+  cy.get('cx-tab-paragraph-container > h3')
+    .first()
+    .should('contain', 'ALL ORDERS')
+    .click();
+}
+
+export function cancelOrder() {
+  cy.get('cx-order-history a.cx-order-history-value')
+    .contains('In Process')
+    .click({ force: true });
+
+  cy.get('cx-order-details-actions button.btn-primary').click({ force: true });
+
+  cy.get('button.cx-counter-action')
+    .contains('+')
+    .click({ force: true });
+
+  cy.get('button.btn-primary')
+    .contains('Continue')
+    .click({ force: true });
+
+  cy.get('cx-cancel-order-confirmation button.btn-primary')
+    .contains('Submit Request')
+    .click({ force: true });
+
+  cy.url().should('contain', 'my-account/orders');
```

Comment:
similar to this https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5843/files#r364932018

but for the order history page

Reply:
add a check for the banner message that displays when successfully cancelling it
![Screen Shot 2020-01-10 at 1 53 11 PM](https://user-images.githubusercontent.com/29099466/72178264-9711ac80-33b0-11ea-91ab-948ed05db52e.png)